### DISCUSSION TITLE
Attempting to use Python module to crate implants.

### DISCUSSION THREAD

**Discussion by DrNickolaosKoroniotis** [BishopFox/sliver#665] [Q&A]
Hello, 

I am experimenting with the provided python interface to Sliver, and I am finding it a bit hard to generate an implant, after I have connected to the Sliver server. 
So, after I have connected to the server with SliverClient, there is a module named "generate" that takes as argument an ImpantConfiguration. 

My issue is that I am unsure how to user ImplantConfiguration in this case to specify type of file to generate, communication protocol for the target-to-C2 connection and output directory. 

Any help (preferably with a brief example) would be much appreciated.

**Comment by moloch--** [comment]
The implant configuration can be fairly complex, and we don't have any wrappers for it in the Python library. The best way to figure out the api is probably to look at the golang/client implementation.
